src/main/java/uk/gov/hmcts/reform/prl/enums/ServeOrderFieldsEnum.java

```diff
@@ -3,14 +3,12 @@
 public enum ServeOrderFieldsEnum {
 
     serveOrderAdditionalDocuments("serveOrderAdditionalDocuments"),
-    isCafcass("isCafcass"),
```

Comment:
isCafcass also needs to be cleared?

Reply:
It was cleared earlier, no need to clear now